js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -310,6 +321,21 @@ const createIndicesHelper =
         return rank < 2 ? varIndices : `i2o_${name}(${varIndices})`;
       };
 
+      let broadcastedIndicesToOffsetImplementation = '';
+      const broadcastedIndicesToOffset = (varIndices: string, output: IndicesHelper) => {
```

Comment:
if user call this multiple times, the behavior could be wrong.

there are 2 ways to resolve this:
- change `broadcastedIndicesToOffsetImplementation` to a map so that different outputs can be stored separately.
- simply check if `broadcastedIndicesToOffsetImplementation` is not empty, throw error saying not support multiple outputs for a indices helper instance and put a TODO there if in future we need.

Reply:
Done，PTAL